src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RemoveAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,201 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Globalization;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Backup.Model;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Sql.Database_Backup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlDatabaseLongTermRetentionBackup", DefaultParameterSetName = RemoveBackupDefaultSet, SupportsShouldProcess = true)]
+    public class RemoveAzureRmSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseLongTermRetentionBackupCmdletBase
+    {
+        /// <summary>
+        /// Parameter set name for the default remove.
+        /// </summary>
+        private const string RemoveBackupDefaultSet = "RemoveBackupDefault";
+
+        /// <summary>
+        /// Parameter set name for remove with an input object.
+        /// </summary>
+        private const string RemoveBackupByInputObjectSet = "RemoveBackupByInputObject";
+
+        /// <summary>
+        /// Parameter set name for remove with a resource ID.
+        /// </summary>
+        private const string RemoveBackupByResourceIdSet = "RemoveBackupByResourceId";
+
+        /// <summary>
+        /// Gets or sets the name of the location the backup is in.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = RemoveBackupDefaultSet,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
remove valuefrompipelinebypropertyname from location, servername, and databasename

Reply:
Not addressed - please remove.

Reply:
Done.